Packs/NetscoutAED/Integrations/NetscoutAED/NetscoutAED.py

```diff
@@ -0,0 +1,1089 @@
+                                      headers=['url', 'pgid', 'cid', 'update_time', 'annotation'],
+                                      headerTransform=string_to_table_header, removeNull=True)
+
+    return CommandResults(
+        outputs_prefix='NetscoutAED.Inbound.Blacklist.Url',
+        outputs_key_field='url',
+        outputs=urls_list,
+        raw_response=raw_result,
+        readable_output=readable_output
+    )
+
+
+def handle_url_addition_commands(client: Client, demisto_args: dict) -> CommandResults:
+    """
+
+        Adds the urls to the inbound blacklisted list.
+
+        :type client: ``Client``
+        :param client: Client to use.
+
+        :type demisto_args: ``dict``
+        :param demisto_args: The demisto arguments.
+
+        :return: The command results which contains the added urls to the inbound blacklisted list.
+        :rtype: ``CommandResults``
+
+    """
+    url = demisto_args.get('url')
+
+    if not url:
+        raise DemistoException(
+            "You must provide url in order to add it to the inbound blacklisted list.")
+
+    remove_nulls_from_dictionary(demisto_args)
+    demisto_args = camelize(demisto_args, "_", upper_camel=False)
+    demisto_args['url'] = ','.join(argToList(url))
+
+    raw_result = client.inbound_blacklisted_url_add_command(demisto_args)
+    urls_list = copy.deepcopy(raw_result.get('urls', [raw_result]))
+    msg = "Urls were successfully added to the inbound blacklisted list"
+
+    objects_time_to_readable_time(urls_list, 'updateTime')
+
+    readable_output = msg + '\n' + tableToMarkdown('Added Urls', urls_list,
+                                                   headers=['url', 'pgid', 'cid', 'update_time', 'annotation'],
+                                                   headerTransform=string_to_table_header, removeNull=True)
+
+    return CommandResults(
+        outputs_prefix="NetscoutAED.Inbound.Blacklist.Url",
+        outputs_key_field="url",
+        outputs=urls_list,
+        raw_response=raw_result,
+        readable_output=readable_output
+    )
+
+
+def handle_url_deletion_commands(client: Client, demisto_args: dict) -> str:
+    """
+
+        Removes urls from the inbound blacklisted list.
+
+        :type client: ``Client``
+        :param client: Client to use.
+
+        :type demisto_args: ``dict``
+        :param demisto_args: The demisto arguments.
+
+        :return: A message which says that the urls were successfully deleted from the list.
+        :rtype: ``str``
+
+    """
+    url = demisto_args.get('url')
+
+    if not url:
+        raise DemistoException("You must provide url in order to remove it from the inbound blacklisted list.")
+
+    remove_nulls_from_dictionary(demisto_args)
+    demisto_args = camelize(demisto_args, "_", upper_camel=False)
+    demisto_args['url'] = ','.join(argToList(url))
+
+    raw_result = client.inbound_blacklisted_url_remove_command(demisto_args)
+    if not raw_result.status_code == 204:
```

Comment:
Checking for status code 204 is the way for me to know that the deletion was executed properly since I am not getting anything informative from the response. I think that any other status code that I can get will represent an error in the api call and will be caught in the main's try/except section. What do you suggest to do?